terminal/tcell/event.go

```diff
@@ -17,7 +17,7 @@ package tcell
 import (
 	"image"
 
-	"github.com/gdamore/tcell"
+	"github.com/gdamore/tcell/v2"
```

Comment:
Would you mind specifically naming this as "tcell"?

```go
tcell "github.com/gdamore/tcell/v2"
```

I use goimports to organize the imports and without the rename goimports just removes the import altogether, since it doesn't see "v2" being used in the code. Maybe also be a bug in the goimports version I use? Do you know?

In case we change this, we should do so in all the locations.

Reply:
I'm using whatever the defaults are in vscode and I don't have that issue.

Reply:
That suggests this will be problem with my setup. Thanks for renaming it for now.